demisto_sdk/commands/validate/validators/RN_validators/RN114_validate_release_notes_header.py

```diff
@@ -0,0 +1,252 @@
+from __future__ import annotations
+
+import re
+from typing import Any, Dict, Iterable, List, Set, Tuple, Union
+
+from demisto_sdk.commands.common.constants import (
+    RN_HEADER_BY_FILE_TYPE,
+)
+from demisto_sdk.commands.common.logger import logger
+from demisto_sdk.commands.content_graph.common import (
+    ContentType,
+)
+from demisto_sdk.commands.content_graph.objects.content_item import ContentItem
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Pack
+RN_HEADER_BY_CONTENT_TYPE = {
```

Comment:
instead of maintaining this dict, let's add to ContentType a property method `release_note_header()` that returns something like `pascalToSpace(self.as_folder)`, and handle manually special cases like `Mappers`, `PreProcess Rules`, `Triggers Recommendations`